packages/p2p/src/components/my-ads/buy-ad-payment-methods-list.jsx

```diff
@@ -11,15 +11,25 @@ const BuyAdPaymentMethodsList = ({ selected_methods, setSelectedMethods, touched
     const { my_ads_store, my_profile_store } = useStores();
     const [selected_edit_method, setSelectedEditMethod] = React.useState();
     const [payment_methods_list, setPaymentMethodsList] = React.useState([]);
+    const [is_close_circle_icon_showing, setIsCloseCircleIconShowing] = React.useState(false);
+    const [show_list, setShowList] = React.useState(false);
+    const [hide_list, setHideList] = React.useState(false);
+    const deleted_autocomplete_ref = React.useRef();
 
     const MAX_PAYMENT_METHOD_SELECTION = 3;
 
+    useOnClickOutside(deleted_autocomplete_ref, () => {
+        setShowList(false);
+        setHideList(true);
+        my_ads_store.setCurrentMethod({ key: null, is_deleted: false });
+    });
```

Comment:
had to use useOnClickOutside instead of onBlur for the deleted autocomplete, as the component isn't focused/clicked on when it appears, thus onBlur doesn't know the component is currently in focus. So I needed this hook to help hide the dropdown list if the user hasn't clicked on the component